peopleops/index.md

```diff
@@ -20,9 +20,39 @@ Before you can take time off you should _always_:
 
 ## Expenses
 
-Expenses can be added on [work.pilot.co/expenses](https://work.pilot.co/expenses)
+The company will cover any reasonable work-related expense, including ensuring you
+have the equipment (laptop/monitor/accessories) you need to do the job. As an
+all-remote company, we meet via Zoom. As such, we will cover the cost of a good
+webcam/microphone/lighting arrangement.
+
+If you do require any equipment, talk to your manager to discuss options.
+
+The general rule is: treat it like your own money - seek good value and use 
+common sense.
+
+There are two ways of spending company money - either using a company credit card,
```

Comment:
Can we swap the credit card and expensing? I don't think everyone should have a credit card, and we should prefer expensing.

Reply:
I disagree, expensing is asking employees to extend to the company a line of credit, reimbursement should be for edge cases where the employee couldn't use their company card for some reason.
We should be able to trust employees with a company credit card to be sensible and follow the policy, if they don't then there's a bigger problem.

Reply:
> expensing is asking employees to extend to the company a line of credit

I don't see any problem with this. It's for two weeks on average, large expenses can always be put on a card when we're adding them to the excell sheet.

> We should be able to trust employees with a company credit card to be sensible and follow the policy

This is not a trust issue for me, but limiting abilities and scope for security. Much like you're (hopefully) never `root` on your system, but you're running `sudo`.

What about:
- Expenses up to $500,-- are expensed
- Else you use the CC if you have one, or ask your manager to buy it for you

Reply:
root vs sudo is more like having a single shared card that everyone knows the details for vs having your own individual cards. You still trust the employees with the level of access they have as sudo its just that you can ensure an indiviual is accountable.
While 2 weeks is fine for some people and they have the ability to get a credit card in their own name we shouldn't assume that will apply to everyone, some people may have personal issues that make it hard for them to get a credit card and feel embarrassed about having to be the one person in a team that can't afford to sub their employer.

Reply:
Thanks for the discussion. Having bounced it around, let's stick with the wording as proposed. There is an open question of whether we automatically provide a brex card to everyone when they join or not. That can be a discussion between the employee and their manager.

And of course, all of this can be reviewed and iterated upon based on experience and feedback.